### ISSUE TITLE
[bug] whitescreen on Arch/KDE Plasma

### ISSUE THREAD

**Issue by T0jan** [bakustarver/rpgmakermlinux-cicpoffs#25]
Hi, 

I have a relatively fresh install of Arch with KDE Plasma running on my system and tried to run some MV games with this tool. But sadly it won't work. If I try to start the game (or the options) from the context menu, literally nothing happens. If I try to start from terminal I get a window with a blank white screen and in the terminal the error message:

```
[1230/021147.791548:ERROR:elf_dynamic_array_reader.h(64)] tag not found
```

I did a quick online check and it seems to be an issue related to chromium based browsers usually. I tried if installing chromium would help but nothing changed. As said it's a pretty fresh Arch install so I may lack some packages which are required but not included in the installer. Another idea I had was if maybe wayland was to blame?

**To Reproduce**
1. Go to the game directory either in file browser or terminal
2. Try to start the game
3. Nothing happens/See error

**Expected behavior**
Game should start

**Screenshots**

![image](https://github.com/user-attachments/assets/bbf75ede-6ad1-4e84-9414-1a4f53319976)

**Desktop (please complete the following information):**
 - OS: Operating System: Arch Linux 
KDE Plasma Version: 6.2.4
KDE Frameworks Version: 6.9.0
Qt Version: 6.8.1
Kernel Version: 6.12.6-arch1-1 (64-bit)
 - Browser firefox (chromium)
 - Version 133.0.3 (131.0.6778.204)

**Does the game crash at startup?**
If it does, whitescreen

**Do other games launch with the same issue?**
all tested games show the same behavior.

**Are you using the compatibility tool to run in the steam platform?**
No.

**Additional context**

**Comment by bakustarver** [comment]
you're right, most likely missing libraries, I used the default ones, so everything works in Ubuntu and manjaro and fedora.

**Comment by bakustarver** [comment]
I'll write a command to check for missing ones.

**Comment by Cyocaff** [comment]
I have the same error, however I've noticed that games just work once per system reboot, and I've also noticed that in that first run that works the same error appears on the terminal, so I'm doubting the possibility of that elf_dynamic_array_reader being the main problem

**Comment by Zwero1** [comment]
> I have the same error, however I've noticed that games just work once per system reboot, and I've also noticed that in that first run that works the same error appears on the terminal, so I'm doubting the possibility of that elf_dynamic_array_reader being the main problem

Yeah when I launch Omori, it doesnt work, I just get a white screen and the elf_dynamic_array_reader error, but when I try In Stars And Time, I get the same error, except the game works perfectly

**Comment by Nyarlathoteddy** [comment]
Had this issue on GNOME 43.2 (Wayland) on Fedora 41 with the same error. Found a workaround where you can right click the whitescreen and choose Reload to have the game launch properly, so far has worked 100% of the time I've used it. I've been using nwjs .95 SDK when doing this, unaware of if this is helping when i reload it.

**Comment by Sylvie-The-Daft** [comment]
i can vouch for the reloading thing and i can also confirm that you need the sdk version for reloading to work, otherwise right clicking on the window does nothing (i'm on debian 12 xfce if that helps)
